keyboards/kc60/readme.md

```diff
@@ -1,35 +1,19 @@
-KC60 (version 2.0) keyboard firmware
-======================
+# KC60
 
-## Quantum MK Firmware
-For the full Quantum feature list, see [the parent readme.md](/readme.md).
+![KC60](https://raw.githubusercontent.com/noroadsleft/qmk_images/master/keyboards/kc60/kc60.jpg)
 
-## Building
-Download or clone the whole firmware and navigate to the keyboards/kc60 folder. Once your dev env is setup, you'll be able to type `make` to generate your .hex - you can then use the Teensy Loader to program your .hex file.
+A 60% PCB supporting in-switch LEDs, formerly available on AliExpress and (Mass)Drop. Supports ANSI, ISO, split Backspace, and both standard and Tsangan bottom row layouts.
 
-Depending on which keymap you would like to use, you will have to compile slightly differently.
+* Keyboard Maintainer: [The QMK Community](https://github.com/qmk)
+* Hardware Supported: KC60 v1.6, v2.0; ATmega32U4
+* Hardware Availability: [Drop.com](https://drop.com/buy/kc60-mechanical-keyboard?mode=guest_open) (no longer available)
 
-### Default
-To build with the default keymap, simply run `make default`.
+Make example for this keyboard (after setting up your build environment):
 
-### Other Keymaps
-Several version of keymap are available in advance but you are recommended to define your favorite layout yourself. To define your own keymap create file named `<name>.c` in the keymaps folder, and see keymap document (you can find in top readme.md) and existent keymap files.
+    make kc60:default
 
-To build the firmware binary hex file with a keymap just do `make` with a keymap like this:
+## RGB Underglow Support
 
-```
-$ make [default|jack|<name>]
-```
+Build with RGB Underglow support by running `make kc60:ws2812`.
```

Comment:
This keymap is way over the size limit.
How about adding mentioning `RGBLIGHT_ENABLE = yes` instead and adding and `ifdef` to the keyboard's `config.h`?

Reply:
Adding the pin and animation config would work, but the PCB doesn't have built-in pads for the RGB LEDs. That keymap is an example of adding an RGB strip to the board (it's got some pins broken out onto the PCB), so there's no set number of LEDs.

Reply:
I see. Still, I'm not sure if setting some random number and adding an in-line comment would be worse user-experience, than recommending a "broken" keymap.

Reply:
Spent some time thinking about this. Decided to go with 16 LEDs as defined in the ws2812 keymap, and update the keymap's file to use as a reference and starting point.

https://github.com/qmk/qmk_firmware/pull/8782/files/71e287c20e90e0e24aea2b507ef8acd1aaeaf906..81a5cda7fc5252bb4cf6fa32f982bb8ee248cae7